src/detectors/yin.js

```diff
@@ -39,7 +39,6 @@ module.exports = function(config = {}) {
     // Set buffer size to the highest power of two below the provided buffer's length.
     let bufferSize;
     for (bufferSize = 1; bufferSize < float32AudioBuffer.length; bufferSize *= 2);
-    bufferSize /= 2;
```

Comment:
This is done below as well with `const yinBufferLength = bufferSize / 2;`

While I couldn't test too low, this did not seem to cause any issues for me and I was immediately able to detect the low E.

Reply:
It seems like this makes the comment on line 39 incorrect, unfortunately, and that comment came from the paper, so I'm hesitant to make this change